test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java

```diff
@@ -166,7 +166,7 @@ public void testMetrics() throws Exception {
         UtilWaitThread.sleep(100);
         try (TabletsMetadata tm = getCluster().getServerContext().getAmple().readTablets()
             .forLevel(DataLevel.USER).fetch(ColumnType.ECOMP).build()) {
-          count = tm.stream().flatMap(t -> t.getExternalCompactions().keySet().stream()).count();
+          count = tm.stream().mapToLong(t -> t.getExternalCompactions().keySet().size()).sum();
```

Comment:
I will have to look this data structure up again but I thought there was something unique about `flatMap`... Do you know if this translates to the same data structure?

Reply:
The output of the user function that you pass to flatMap should be a Stream. So, you're converting the original stream into a stream of streams. flatMap then just concatenates them, so all the elements of each of those streams are put into one output stream.

These should be identical. Essentially, it goes from `sizeof(concat(set1, set2, ... setN))` to `sum(sizeof(set1), sizeof(set2), ... sizeof(setN))`.